testing/distributions-integ-tests/src/integTest/groovy/org/gradle/SrcDistributionIntegrationSpec.groovy

```diff
@@ -53,6 +53,7 @@ class SrcDistributionIntegrationSpec extends DistributionIntegrationSpec {
     def sourceZipContents() {
         given:
         TestFile contentsDir = unpackDistribution()
+        // TODO Need to update src distribution to include "packaging"
```

Comment:
❓ Does this indicate that the change to `build-logic/packaging/src/main/kotlin/gradlebuild/packaging/GradleDistributionSpecs.kt` is not enough yet?

Reply:
Forgot to remove the todo, this is done